pkg/network/setup/hotplugcontroller_test.go

```diff
@@ -0,0 +1,49 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/tests/libvmi"
+)
+
+var _ = Describe("NetworksToHotplug", func() {
+	const (
+		nadName     = "nad1"
+		networkName = "n1"
```

Comment:
If this is not confusing, I do not know what is.

Reply:
The argument that we need to keep a bad naming API convention, and now to see it in action.
	nadName        = "nad1"
		networkName    = "n1"

Before, we had `networkName` as pointing to the NAD name. Now it points to the network/interface name.
And the `nadName` points to the real NAD name.
Looking at the usage, we see: `NetworkName: nadName` and `Name: networkName`.
Same name, once as a field name and once as a variable name, but not ref to the same thing. My brain crashed.

Reply:
ok, I understand your brain crashed. 

What action can I do to prevent it from happening again  ? 

What variable - and values - do you suggest to make this more palatable ?

Reply:
That we will be consistent with using nadName/netAttachDefName for NAD things and networkName for VMI spec networks and interfaces.
If the function name is explicit about touching the VMI network or interface, the argument there can be just `name` for the logical name to avoid confusion.

That would be my preferred rule-of-thumb.
And yes, it does not solve the whole brain crash, because the API naming is not good.

I think you can resolve this.